src/screens/DiscoveryScreenV2/hooks/useDiscovery.js

```diff
@@ -21,17 +26,43 @@ const useDiscovery = () => {
       });
 
       await getFollowingTopic().then((response) => {
-        console.log({response}, 'likak');
         following.setFollowingTopics(response.data, followingDispatch);
       });
       setRefreshing(false);
     } catch (e) {
       setRefreshing(false);
     }
   };
+
+  const onRefreshDiscoveryUser = async () => {
+    try {
+      const discoveryInitialUserResponse = await DiscoveryRepo.fetchInitialDiscoveryUsers();
+      DiscoveryAction.setDiscoveryInitialUsers(
+        discoveryInitialUserResponse.suggestedUsers,
+        discoveryDispatch
+      );
+    } catch (e) {
+      setRefreshing(false);
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There is a potential issue with error handling in the `onRefreshDiscovery` and `onRefreshDiscoveryUser` functions. If an error occurs, you are setting `refreshing` to `false`, but there is no other error handling mechanism. It would be beneficial to have some way of notifying the user or logging the error for debugging purposes.
```diff
     } catch (e) {
       setRefreshing(false);
+       console.error(e);
     }
   };
...
    } catch (e) {
      setRefreshing(false);
+       console.error(e);
    }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->